ocean_lib/models/data_token.py

```diff
@@ -262,12 +262,11 @@ def verify_order_tx(self, web3, tx_id, did, service_id, amount_base, sender):
             target_amount = target_amount - order_log.args.marketFee
 
         # verify sender of the tx using the Tx record
-        tx = web3.eth.getTransaction(tx_id)
         if sender not in [order_log.args.consumer, order_log.args.payer]:
             raise AssertionError(
                 "sender of order transaction is not the consumer/payer."
             )
-
+        tx = web3.eth.getTransaction(tx_id)
```

Comment:
why is this added here? I see no usage of it later.

Reply:
Saved some wrong modification that time. Now refactored it.